addon/utils/slugify.js

```diff
@@ -3,5 +3,5 @@ import slugify from "slugify";
 export default function(value) {
   return slugify(value.toLowerCase(), {
     remove: /((?![a-z0-9-\s]).)/g
-  }).substr(0, 50);
+  }).substr(0, 150);
```

Comment:
Why 150 instead of 255?

Reply:
Because the backend hast 150.

Reply:
Yes obviously, but why? I wasn't assigned in the backend PR so I'm asking now.

I'm just a bit concerned that we'll have to do this again as soon as 150 is not enough. So since the DB would allow 255 as a maximum I was wondering if it would be better to use the absolute maximum...

Reply:
This is not the right place for this discussion. If you prefer a longer max-length, open a PR on the backend.